examples/data/quiz_multichoice_answers.csv

```diff
@@ -0,0 +1,41 @@
+id,question_id,text,grade,feedback
+0,1a7b77c1-e9e1-4f22-ba84-56a74b733419,<p>5</p>,0.0,nope
+1,1a7b77c1-e9e1-4f22-ba84-56a74b733419,<p>6</p>,0.0,nah
+2,1a7b77c1-e9e1-4f22-ba84-56a74b733419,<p>11</p>,1.0,yes
+3,1a7b77c1-e9e1-4f22-ba84-56a74b733419,<p>13</p>,0.0,no
+4,73cf891e-1441-433c-a00e-45fc723b0996,<p>20</p>,0.0,no
+5,73cf891e-1441-433c-a00e-45fc723b0996,<p>16</p><,1.0,sure
+6,73cf891e-1441-433c-a00e-45fc723b0996,<p>5</p><,0.0,not quite
+7,73cf891e-1441-433c-a00e-45fc723b0996,<p>90</p></div>,0.0,no
+8,8a5008a7-970b-411b-9ed7-d1970b2bed56,<p></p>,0.0,no
+9,8a5008a7-970b-411b-9ed7-d1970b2bed56,<p>80</p><,0.0,nope
+10,8a5008a7-970b-411b-9ed7-d1970b2bed56,<p>0</p>,1.0,yes
+11,8a5008a7-970b-411b-9ed7-d1970b2bed56,<p>234</p>,0.0,no
+12,9368abf2-5a6c-4254-ab57-3edfdb338511,<p>4</p>,0.0,no
+13,9368abf2-5a6c-4254-ab57-3edfdb338511,<p>88</p>,0.0,no
+14,9368abf2-5a6c-4254-ab57-3edfdb338511,<p>9</p>,1.0,yassssss
+15,9368abf2-5a6c-4254-ab57-3edfdb338511,<p>99</p>,0.0,nuh uh
+16,574085b1-06f0-485b-9fe6-a784ba37e981,<p>20</p>,1.0,YUP
+17,574085b1-06f0-485b-9fe6-a784ba37e981,<<p>234</p>,0.0,no
+18,574085b1-06f0-485b-9fe6-a784ba37e981,<p>49</p>,0.0,no
+19,574085b1-06f0-485b-9fe6-a784ba37e981,<p>44</p>,0.0,no
+20,0c6d9bee-7e89-4c87-8674-da0d56090b57,<p>20</p>,1.0,YES THAT'S IT
+21,0c6d9bee-7e89-4c87-8674-da0d56090b57,<p>88</p>,0.0,no
+22,0c6d9bee-7e89-4c87-8674-da0d56090b57,<p>9</p>,0.0,no
+23,0c6d9bee-7e89-4c87-8674-da0d56090b57,<p>8</p>,0.0,no
+24,2ffd03b0-67af-4e96-9476-5d1b621f1725,<p>5</p>,0.0,no
+25,2ffd03b0-67af-4e96-9476-5d1b621f1725,<p>77</p>,0.0,no
+26,2ffd03b0-67af-4e96-9476-5d1b621f1725,<p>6</p>,0.0,no
+27,2ffd03b0-67af-4e96-9476-5d1b621f1725,<p>16</p>,1.0,yes
+28,99134f44-0590-4fd3-86af-c926785b4a32,<p>8</p><,0.0,no
+29,99134f44-0590-4fd3-86af-c926785b4a32,<p>0</p>,1.0,yes
+30,99134f44-0590-4fd3-86af-c926785b4a32,<p>2</p><,0.0,no
+31,99134f44-0590-4fd3-86af-c926785b4a32,<p>1</p>,0.0,no
+32,da21613b-5d05-48d2-a07b-2bec166e5e54,<p>6</p>,1.0,yes
+33,da21613b-5d05-48d2-a07b-2bec166e5e54,<p>9</p>,0.0,no
+34,da21613b-5d05-48d2-a07b-2bec166e5e54,<p>10</p>,1.0,no
+35,da21613b-5d05-48d2-a07b-2bec166e5e54,<p>11</p>,0.0,no
```

Comment:
@mjoss17 - This may have been a copy / paste error, but I randomly noticed that this question has two answers with full credit.